find-trio-denovo.py

```diff
@@ -14,167 +13,105 @@
 #############################################################################################################
 def main():
 	startTime = time.time()
-	inFileName = sys.argv[1] 
-	firstCol = sys.argv[2]   # user inputs the order of the parent and child columns
-	secondCol = sys.argv[3]  # so that the program knows which is being read
-	thirdCol = sys.argv[4]
+	inFileName = sys.argv[1]
+        momIdx = sys.argv.index("mom")
+        dadIdx = sys.argv.index("dad")
+        childIdx = sys.argv.index("child")
```

Comment:
This keeps you from having to define how you extract the data from each line. Extract the 3 samples together and then split them based on the index here.